packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code.vue

```diff
@@ -356,16 +363,16 @@ async function handleCode() {
 			}
 		};
 
-		const modelId = await codeBlocksToAmr(newCode, file);
+		const model: Model | null = await codeBlocksToAmr(newCode, file);
```

Comment:
`model: Model | null` ❤️

Reply:
That really hurts my brain, it does NOT read correctly for me - I assume it is model should be whatever the function returns or null, but I see it as either model or null gets assigned the function's value which is insane